### PR TITLE
#162163256 TravisCI readme badge

### PR DESCRIPTION
**Pull request by joshtrigger**
**What does this PR do?**
Have a TravisCI readme badge

**Description**
This readme badge will display in percentage how much of the code base is e=being covered by the tests.

**What are the relevant Pivotal tracker stories?**
#162163256

### CODE REVIEW

**Comment by Kasulejoseph** [comment]
good work @joshtrigger

**Review by Kasulejoseph — APPROVED** [review]

**Review by Nta1e — APPROVED** [review]

**Review by SokoPaulSokool — APPROVED** [review]

**Review by bibangamba — APPROVED** [review]

File: authors/settings.py [hunk 1]
```diff
@@ -78,6 +78,19 @@
 # Database
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
+if 'TRAVIS' in os.environ:
+    DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'athenatraviscidb',
+        'USER': 'postgres',
```
Comment (collin5) [inline authors/settings.py]: Can you please pick these from the `environment`. https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings

**Review by collin5 — CHANGES_REQUESTED** [review]

File: authors/settings.py (hunk 1 above)
Comment (joshtrigger) [reply authors/settings.py]: I have worked on the issue

**Review by collin5 — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**